internal/pkg/skuba/kubernetes/versions.go

```diff
@@ -92,7 +92,7 @@ var (
 			},
 			AddonsVersion: AddonsVersion{
 				Cilium:  &AddonVersion{"1.5.3", 1},
-				Kured:   &AddonVersion{"1.2.0-rev4", 1},
+				Kured:   &AddonVersion{"1.3.0", 1},
```

Comment:
```suggestion
				Kured:   &AddonVersion{"1.3.0", 2},
```

Whenever we change anything (even the version itself) we should bump the manifest version as well.